src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/Controllers/AccountController.cs

```diff
@@ -153,7 +159,9 @@ public ActionResult Login(string returnUrl)
         /// <returns>Details action result</returns>
         [HttpPost]
         [Authorize]
+#pragma warning disable S4261 // Methods should be named according to their synchronicities
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not suppress issues. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS1309)